Jenkinsfile

```diff
@@ -221,6 +266,104 @@ pipeline {
                 '''
             }
         }
+
+        stage('dist') {
+            steps {
+                sh '''
+                    set -e +x
+                    source ${PWD}/.envrc
+                    set -x
+                    unset HCF_PACKAGE_COMPILATION_CACHE
+                    rm -f scf-*amd64*.zip
+                    make helm bundle-dist
+                '''
+            }
+        }
+
+        stage('deploy') {
+			when {
+				expression { return params.TEST_SMOKE || params.TEST_BRAIN || params.TEST_CATS }
+			}
+            steps {
+                timeout(time: 2, unit: 'HOURS') {
```

Comment:
There's still a 10h global timeout for this pipeline; I assume that's still in effect? Would a wipe build + full (including 🐱 ) test run + source tarring blow the budget?

Reply:
Fixed.